evansmusomi/burger-builder
# Enables Users to Customize and Preview their Burger

*What this does*
- Renders burger component
- Adds burger build controls to add or remove ingredients
- Ensures user cannot remove an ingredient that isn't added

*How to verify*
- User should be able to preview built burger
- User should see a prompt to add ingredients if none
- User should be able to add or remove an ingredient
